django/db/models/options.py

```diff
@@ -690,6 +690,24 @@ def get_path_to_parent(self, parent):
                 path.append(PathInfo(final_field.model._meta, opts, targets, final_field, False, True))
         return path
 
+    def get_path_from_parent(self, parent):
+        """
+        Get path from parent model to current model.
+
+        Returns an empty list if parent is not a parent of the current model.
+        """
+        if self.model is parent:
+            return []
+        model = self.concrete_model
+        chain = model._meta.get_base_chain(parent)
+        chain.reverse()
+        chain.append(model)
+        path = []
+        for i in range(len(chain) - 1):
```

Comment:
Some inlines comments? 
Not sure if something like `for i, obj in enumerate(chain[:-1])` might be a better formulation of this logic.